extensions/common/json_util.h

```diff
@@ -15,6 +15,9 @@
 
 #pragma once
 
+#include <string>
+#include <utility>
```

Comment:
Curious, what is this for? Does it substantially increase generated wasm file size?

Reply:
It was not needed, destroyed.